Documentation/debugging/unix-instructions.md

```diff
@@ -65,13 +65,16 @@ lldb-3.9 -O "settings set target.exec-search-paths /home/parallels/Downloads/Sys
 
 - Install [Visual Studio Code](https://code.visualstudio.com/)
 - Install the [C# Extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp)
-- Open the folder containing the source you want to debug in VS Code
+- Open the folder containing the source you want to debug in VS Code - i.e., if you are debugging a test failure in System.Net.Sockets, open `corefx/src/System.Net.Sockets`
 - Open the debug window: `ctrl-shift-D` or click on the button on the left
 - Click the gear button at the top to create a launch configuration, select `.NET Core` from the selection dropdown
 - In the `.NET Core Launch (console)` configuration do the following
   - delete the `preLaunchTask` property
-  - set `program` to the full path to corerun in the test directory
-  - set `cwd` to the test directory
+  - set `program` to the full path to `dotnet` in the bin directory.
+    - something like `corefx/bin/testhost/netcoreapp-Linux-{Configuration}-{Architecture}`, plus the full path to your corefx directory.
+  - set `cwd` to the test bin directory.
+    - using the System.Net.Sockets example, should be something like `corefx/bin/tests/System.Net.Sockets.Tests/netcoreapp-Linux-{Configuration}-{Architecture}`, plus the full path to your corefx directory.
```

Comment:
Nit: it should be

Reply:
It should be what?

Reply:
"using the System.Net.Sockets example, should be something" =>
"using the System.Net.Sockets example, it should be something"

Reply:
Oh, gotcha. Will change